.github/issue_template.md

```diff
@@ -0,0 +1,40 @@
+Note: This is just a template, so feel free to use/remove the unnecessary things
+
+### Description
+- Type: Bug | Enhancement\Feature Request | Question
+- Priority: Blocker | Major | Minor
+
+---------------------------------------------------------------
+## Bug
+
+**OS**  
+linux|windows|??
+
+**mbed TLS build:**  
+Version: x.x.x or git commit id  
+Configuration: please attach config.h file  
+Compiler and options (if you used a pre-built binary, please indicate how you obtained it):  
+Additional environment information:  
+
+**peer device TLS stack and version**  
+openSSL | GnuTls | other  
+version:  
+
+**Expected behavior**   
+
+**Actual behavior**  
+
+**Steps to reproduce**  
+
+----------------------------------------------------------------
+## Enhancement\Feature Request
+
+**Incentive for change**  
```

Comment:
I think I'd prefer something simpler than 'Incentive for change'. How about 'Justification - why does the library need this feature?'.